apps/native-component-list/src/screens/AV/VideoPlayer.tsx

```diff
@@ -151,7 +151,12 @@ function ResizeModeSegmentedControl({
         onChange={event => {
           setIndex(event.nativeEvent.selectedSegmentIndex);
         }}
-        onValueChange={value => onValueChange(resizeMap[value])}
+        onValueChange={value => {
+          const mappedValue = resizeMap[value];
+          if (mappedValue) {
+            onValueChange(mappedValue);
+          }
+        }}
```

Comment:
If TS is complaining here then you need to adjust it somewhere else. From what I can see here, `value` can only one of known strings (namely, keys from `resizeMap`)
```suggestion
        onValueChange={value => onValueChange(mappedValue)}
```